swagger.yaml

```diff
@@ -2500,6 +2500,105 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/operators:
+    get:
+      tags:
```

Comment:
What does it mean when API belong to two tags? will there be two interfaces that need to implement it?

Reply:
It just groups the operation to multiple tags. No other implementation required.